includes/Core/Dashboard_Sharing/Activity_Metrics/Active_Consumers.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Dashboard_Sharing\Active_Consumers
+ *
+ * @package   Google\Site_Kit\Core\Dashboard_Sharing
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Dashboard_Sharing;
+
+use Google\Site_Kit\Core\Storage\User_Setting;
+
+/**
+ * Class for representing active consumers for an access token.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Active_Consumers extends User_Setting {
```

Comment:
> The meta value should allow for an array of associative arrays to be stored, each with id (integer) and roles (array of strings) keys.

We know the structure of values that will be stored in this setting, let's add the `get_sanitize_callback` method to verify it. Plus, we will need to create a phpunit test to verify that the sanitization callback works as expected.